xbmc/pvr/PVRDatabase.cpp

```diff
@@ -488,36 +490,17 @@ int CPVRDatabase::Get(CPVRChannelGroup &group)
 
 bool CPVRDatabase::PersistChannels(CPVRChannelGroup &group)
 {
-  bool bReturn(true);
+  for (auto &member : group.m_members) {
+    const CPVRChannelPtr &channel = member.second.channel;
```

Comment:
Didn't think that much about it, I just wanted to make the code more readable. Do you want me to drop the `&` or the whole commit?